features/admin.rules.v1/components/rule-conditions.tsx

```diff
@@ -434,34 +462,54 @@ const RuleConditions: FunctionComponent<RulesComponentPropsInterface> = ({
         );
     };
 
-    const ConditionOrDivider = (props: any) => (
-        <Divider sx={ { mb: 1, mt: 2 } }>
-            <Button
-                size="small"
-                variant="contained"
-                color="secondary"
-                onClick={ () => addNewRuleCondition(props.ruleId, props.id, ConditionTypes.Or) }
-                startIcon={ <AddIcon /> }
-            >
-                Or
-            </Button>
-        </Divider>
-    );
-
     return (
```

Comment:
We may need to add a placeholder for this component incase `ruleConditions` are empty or in suspense.

```
if (!ruleConditions) {
    return <Loading />;
}

return (
  <>
    // rest
  </>
);
```

Reply:
Yes, need to add it.